cronenbroguelike/game.py

```diff
@@ -103,6 +103,10 @@ def main():
     game_config = _load_config()
     if game_config.get("extra_commands"):
         from . import extra_commands
-    _start_game(None, game_config)
-    adventurelib.say("")  # Necessary for space before first prompt.
-    adventurelib.start()
+    while True:
+        try:
+            _start_game(None, game_config)
```

Comment:
`_start_game` only needs one arg, change the function signature pls

Reply:
This was not resolved.

Reply:
> This was not resolved.
> 
> https://media.tenor.com/images/c498da35e38a5787114f8c1594e96951/tenor.gif
> 
> woop woop woop

https://media.tenor.com/images/c498da35e38a5787114f8c1594e96951/tenor.gif

woop woop woop

Reply:
Fixed.